app/controllers/test/components/track_lists/tracks_controller.rb

```diff
@@ -0,0 +1,42 @@
+class Test::Components::TrackLists::TracksController < ApplicationController
```

Comment:
I'd really like this to be a a `tracks` action on `Test::Components::TrackLists` please :)

As a secondary general question, when the API is done, will we remove this action altogether and just use factories and the real API?